### DISCUSSION TITLE
How to crash your chromebook (and maybe trick your friends)

### DISCUSSION THREAD

**Discussion by S-PScripts** [3kh0/ext-remover#1260] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
**_How to crash your chromebook (and maybe trick your friends)_**

Requirements:
-> Make sure you have access to chrome://flags

Instructions:
1. Open Chrome.
2. Go to chrome://flags
3. Enable the toolbar pinning flag and restart.
4. Open Chrome again.
5. Click the 3 dots icon at the top right.
6. Click More tools.
7. Click Customize Chrome.
8. Find the Toolbar section in the sidebar that pops up and click it.
9.  In the navigation section, enable New incognito window.
10. Click the incognito  icon that appears at the top right.
11. Crash... profit? What did you expect? You really thought I found a new incognito exploit?

**Comment by schoolexploitkid** [comment]
i cant find customize chrome :vvvvv

**Reply by S-PScripts** [reply]
Toolbar pinning
Enables the ability to pin features to the toolbar via the Customize Chrome side panel. – Mac, Windows, Linux, ChromeOS, Lacros

[#toolbar-pinning](chrome://flags/#toolbar-pinning)

**Reply by S-PScripts** [reply]
> i cant find customize chrome :vvvvv

you need this flag, forgot to mention that

**Comment by YoiseAndZane** [comment]
No wonder my county removed chrome flags

**Comment by jazzymakesstuff** [comment]
chrome://crash

**Reply by 332677-afk** [reply]
lool

**Comment by 2027aferraradifranco-glitch** [comment]
what the hell are you guys saying
